docs/assignment1.md

```diff
@@ -1 +1,53 @@
+
+[1] https://en.wikipedia.org/wiki/Single-elimination_tournament
+
+[2] https://en.wikipedia.org/wiki/Fantasy_football_(association)#Points_scoring
+
+## Features
+
+### Functional features
+Author(s): `Teona Moga`
+
+The functional requirements below represent our understanding of the technical problem to be solved. We defined the constraints that the system needs to adhere to as well as the expectations for its anticipated behaviour.
+
+| ID  | Short name  | Description  |
+|---|---|---|
+| F1  | User-provided data | Allow the user to manually provide player data via the user interface in CSV format. Each row should only contain two values, i.e. name and score, e.g. `Ben Foster,51`. The file shouldn’t have a header row. |
+| F2  | Erroneous data  | In the event user input cannot be processed, present an error detailing the issue to the user.|
+| F3  | Team formation  | Allow the user to form a team of 11 players from the players provided, with no duplicates.|
+| F4  | Multiple teams | System will support the creation of multiple teams. In the case the user only creates one team, random teams are created by the system to reach the minimal number of teams in a league.|
+| F5  | League formation | Allow the user to set up a league with its own schedule. The first round can have at most 16 matches. Matches will be played according to this schedule and the system will output the winner of each individual game. |
+| F6 | Team score | The final score of a team is computed by summing the points scored by each of the players according to the user provided data. |
+| F7 | Soccer match | A soccer match can be simulated between two teams. The winner is chosen based on each team’s final score as described in F5 - the highest scoring team wins. |
+| F8 | Equal scores | In the event two teams have equal scores, the winner is chosen at random. |
+
+### Quality requirements
+Author(s): `Patricia Santana Vasquez`
+
+The following quality requirements were chosen in order to improve ease of use from the user’s standpoint. They were also chosen in order to detail essentials for the team to adhere to for the maintenance of a properly and efficiently functioning system.
+
+| ID  | Short name  | Quality attribute | Description  |
+|---|---|---|---|
+| QR1  | Intuitive UI | Usability  | UI should be simple, intuitive to use and consistent throughout the system. Interface elements should be minimal and allow only the necessary operations the user requires. |
+| QR2  | Instantaneous results| Responsiveness  | Once a league starts, the results of the virtual matches shall be available within 1 second. |
+| QR3  | Documentation | Maintainability  | The code is readable and follows a logical structure. |
+| QR4  | Efficiency | Resource Utilization | Java libraries are used effectively in order to improve the performance of the system. |
```

Comment:
**[SEVERE]** Unverifiable, vague and unclear, remove this QR. Resource utilization is not part of the quality attributes mentioned in lecture 1, stick to those!